app/js/lib/foxbox/foxbox.js

```diff
@@ -67,6 +67,8 @@ export default class Foxbox extends Service {
     this[p.settings] = new Settings();
     this[p.db] = new Db();
     this[p.net] = new Network(this[p.settings]);
+
+    this.boxes = [];
```

Comment:
That's the tiny thing absence of which was making my life harder :) When app is run for integration test, there is a race, app tries to connect to the remote registration service and fails, then login screen fails (boxes is undefined). And only then test modifies local storage to switch to local registration service, but login screen is already broken :/